rotkehlchen/api/v1/schemas.py

```diff
@@ -2460,3 +2462,74 @@ def make_ethereum_transaction_query(  # pylint: disable=no-self-use
         return {
             'filter_query': filter_query,
         }
+
+
+class CustomAssetsQuerySchema(DBPaginationSchema, DBOrderBySchema):
+    name = fields.String(load_default=None)
+    started = TimestampField(load_default=None)
+    swapped_for = AssetField(load_default=None)
+    identifier = fields.String(load_default=None)
+    custom_asset_type = fields.String(load_default=None)
+
+    @post_load
+    def make_custom_assets_query(  # pylint: disable=no-self-use
+            self,
+            data: Dict[str, Any],
+            **_kwargs: Any,
+    ) -> Dict[str, Any]:
+        filter_query = CustomAssetsFilterQuery.make(
+            limit=data['limit'],
+            offset=data['offset'],
```

Comment:
You mentioned int the documentation that it is possible to provide `order_by_attributes` and `ascending` but I don't see them here. Am I missing something?